packages/backend/server/src/plugins/copilot/resolver.ts

```diff
@@ -539,18 +540,18 @@ export class CopilotResolver {
         const filename = createHash('sha256')
           .update(uploaded.buffer)
           .digest('base64url');
-        const link = await this.storage.put(
+        const attachment = await this.storage.put(
           user.id,
           workspaceId,
           filename,
           uploaded.buffer
         );
-        options.attachments.push(link);
+        attachments.push({ attachment, mimeType: blob.mimetype });
```

Comment:
If time permits, it is still recommended to add an e2e test to ensure this logic is not broken in future changes.